lib/src/controls/better_player_controls_configuration.dart

```diff
@@ -193,3 +198,5 @@ class BetterPlayerControlsConfiguration {
         enableProgressText: true);
   }
 }
+
+enum PlayerPlatform { IOS, ANDROID }
```

Comment:
Please change PlayerPlatform to PlayerTheme with values: `Material,` `Cupertino`. Please move it to another class.